debian/snapd.refresh.service

```diff
@@ -1,6 +1,8 @@
 [Unit]
 Description=Automatically refresh installed snaps
 After=network.target
+Requisite=snapd.socket
```

Comment:
Would it make sense to use "After" for this as well ? I.e. `After=network.target snapd.socket` ?